### PR TITLE
Minor typos fix and re-organizing some pages

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

File: docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI.md [hunk 1]
```diff
@@ -22,23 +22,29 @@ is too slow for heavy scripting use.
 To use the CLI, you must create an application credentials and set the
 appropriate environment variables.
 
-Navigate to Identity > Application Credentials
+You can download the environment file with the credentials from the OpenStack dashboard.
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI.md:25]: would a link to the OpenStack Dashboard be helpful here?

File: docs/openstack/create-and-connect-to-the-VM/images.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Images
+
+Image compose of a virtual collection of a kernel, operating system, and configuration.
```
Comment (joachimweyl) [inline docs/openstack/create-and-connect-to-the-VM/images.md:3]: composed

File: docs/openstack/create-and-connect-to-the-VM/launch-a-VM.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Navigate: Project -> Compute -> Images.
 ![VM Images](images/vm_images.png)
 
 Click Launch Instance next to the public image of your choice. In the example,
-we chose **ubuntu-21.04-x86_64**.
+we chose **ubuntu-21.04-x86_64**, you can choose any available images.
```
Comment (joachimweyl) [inline docs/openstack/create-and-connect-to-the-VM/launch-a-VM.md:15]: you `may` choose

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 4]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
-        **MobaXterm** is a free application that can be downloaded using [this link](https://mobaxterm.mobatek.net/download-home-edition.html).
-        After downloading, install it like any other normal Windows software.
+- **Enable WSL on your local Windows 10 subsystem for Linux:**
 
-    - **Open MobaXterm and run WSL Linux:**
-        As you open this advanced terminal for Windows 10, WSL installed Ubuntu
-        app will show on the left side panel of it. Double click on that to start
-        the WSL session.
+    The steps given here are straightforward, however, before following them
+    make sure on your Windows 10, you have WSL enabled and have at least Ubuntu
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:52]: on Windows 10, remove `your`

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 5]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
 
-    ![Virt-Manager Select virtio ISO](images/7.7.select_virtion_iso.png)
+Enable the **VirtIO** driver. By default, the Windows installer does not
+detect the disk.
 
-    Make sure the NIC is using the **virtio** Device model as show below:
+![Virt-Manager Disk with VirtIO driver](images/7.1.customize_sata_disk_virtio.png)
 
-    ![Virt-Manager Modify  NIC](images/7.2.customize_nic_virtio.png)
+Add a **New Hardware**  > select **CDROM device** and attach to downloaded
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:131]: replace Add a with `Click` or `Select`

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 6]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
 
-    ![Windows Boot Options](images/7.8.boot_option_win_cdrom_first.png)
+![Virt-Manager Browse virtio ISO](images/7.6.browse_virtio_iso.png)
 
-4. Click "Apply" button and continue with the Windows installation.
+![Virt-Manager Select virtio ISO](images/7.7.select_virtion_iso.png)
 
-    When prompted you can choose "Windows Server 2022 Standard Evaluation
-    (Desktop Experinece)" option as shown below:
+Make sure the NIC is using the **virtio** Device model as show below:
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:142]: as `shown` below

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 7]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
 
-    ![Windows Desktop Installation](images/7.windows_installation_desktop.png)
+![Virt-Manager Modify  NIC](images/7.2.customize_nic_virtio.png)
 
-    ![Windows Custom Installation](images/8.custom_setup.png)
+![Virt-Manager Apply Change on NIC](images/7.3.customize_nic_virtio_apply.png)
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:146]: might be worth adding a step that states to click Apply?

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 8]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
 
-    Load VirtIO SCSI drivers and network drivers by choosing an installation
-    target when prompted. Click **Load driver** and browse the file system.
+Make sure to set proper order of **Boot Options** as shown below so that
+CDROM with Windows ISO is set on the first and Apply the order change.
+After this please begin windows installation clicking on **"Begin Installation"**
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:150]: installation `by` clicking

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 9]
```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
...
+button.
 
-    ![Windows Custom Load Driver](images/9.load_driver.png)
+![Windows Boot Options](images/7.8.boot_option_win_cdrom_first.png)
 
-    ![Browse Local Attached Drives](images/10.browse_driver.png)
+### 4. Continue with the Windows installation
 
-    ![Select VirtIO CDROM](images/11.browse_CDRom_virtio_iso.png)
+Click "Apply" button and continue with the Windows installation
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:157]: this should come before header 4

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md [hunk 10]
```diff
@@ -239,112 +255,123 @@ running the following commands on **Command Prompt (Run as Adminstrator)**:
...
+### 14. How to have Remote Desktop login to your Windows instance
 
-14. Remote Desktop login should work with the FLoating IP associated with the instance:
+Remote Desktop login should work with the Floating IP associated with the instance:
 
-    ![Search Remote Desktop Protocol locally](images/RDP_on_local_machine.png)
+![Search Remote Desktop Protocol locally](images/RDP_on_local_machine.png)
 
-    ![Connect to Remote Instance using Floating IP](images/remote_connection_floating_ip.png)
+![Connect to Remote Instance using Floating IP](images/remote_connection_floating_ip.png)
```
Comment (joachimweyl) [inline docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:369]: Why are we showing the user a failed attempt to log in?

**Review by joachimweyl — CHANGES_REQUESTED** [review]
I have added suggestions as comments that should be reviewed.

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md (hunk 10 above)
Comment (Milstein) [reply docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:369]: I am uploading a new screenshot for this now.

**Comment by Milstein** [comment]
> I have added suggestions as comments that should be reviewed.

File: docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI.md (hunk 1 above)
Comment (Milstein) [reply docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI.md:25]: Added link to : https://stack.nerc.mghpcc.org/dashboard/identity/application_credentials/

File: docs/openstack/create-and-connect-to-the-VM/flavors.md [hunk 11]
```diff
@@ -6,127 +6,44 @@ configuration for a server.
...
+| c1.s2.2xlarge | 92160     | 25                | 20        |
+| c1.s2.4xlarge | 184320    | 25                | 40        |
+| c2.s2.xlarge  | 32768     | 25                | 16        |
+| c2.s2.2xlarge | 65536     | 25                | 32        |
+| c2.s2.4xlarge | 81920     | 25                | 40        |
+
+NERC also supports the most demanding workloads including Artificial Intelligence
+(AI), Machine Learning (ML) training and Deep Learning modeling, simulation, data
+analytics, data visualization, distributed databases, and more. For such demanding
+workloads, the NERC’s GPU based distributed computing flavor is recommended, which
```
[the remark this answers is not in the document]
Comment (Milstein) [reply docs/openstack/create-and-connect-to-the-VM/flavors.md:38]: done adding GPU - based also going to modify to a new branch content!

File: docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md (hunk 7 above)
Comment (Milstein) [reply docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md:146]: I added click "Apply" button before Step #4

File: docs/openstack/create-and-connect-to-the-VM/flavors.md [hunk 12]
```diff
@@ -6,127 +6,44 @@ configuration for a server.
...
+| c1.s2.2xlarge | 92160     | 25                | 20        |
+| c1.s2.4xlarge | 184320    | 25                | 40        |
+| c2.s2.xlarge  | 32768     | 25                | 16        |
+| c2.s2.2xlarge | 65536     | 25                | 32        |
+| c2.s2.4xlarge | 81920     | 25                | 40        |
+
+NERC also supports the most demanding workloads including Artificial Intelligence
+(AI), Machine Learning (ML) training and Deep Learning modeling, simulation, data
+analytics, data visualization, distributed databases, and more. For such demanding
+workloads, the NERC’s GPU - based distributed computing flavor is recommended, which
```
Comment (joachimweyl) [inline docs/openstack/create-and-connect-to-the-VM/flavors.md:38]: GPU-based, no spaces.

File: docs/openstack/create-and-connect-to-the-VM/flavors.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (joachimweyl) [reply docs/openstack/create-and-connect-to-the-VM/flavors.md]: `GPU-based`, no spaces

**Review by joachimweyl — APPROVED** [review]
all changes are complete and look good. I should preface that I did not do the steps myself. As this was already up I assumed the steps were already tested.

**Pull request merged**